Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -172,6 +172,23 @@ def get_asset_details_command(client: Client, args: dict) -> CommandResults:
         raw_response=parsed,
     )
 
+def core_execute_command_command(args: dict) -> None:
```

Comment:
This function should include the whole logic of the command. 
You can split it into smaller functions, as I wrote before.